interfaces/cython/cantera/examples/thermo/isentropic_units.py

```diff
@@ -0,0 +1,65 @@
+"""
+Isentropic, adiabatic flow example - calculate area ratio vs. Mach number curve
+
+Requires: cantera >= 2.5.0, matplotlib >= 2.0
+"""
+
+import cantera.units as ct
+import math
```

Comment:
I think you can remove this import, and if the same import occurs in the other examples you created, you can probably remove it there too. You're using `numpy` for the `sqrt` now, because of the requirements of Pint.